.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Feature Request
+    url: https://forem.dev/t/features
```

Comment:
Maybe talk about adding the selfhost tag instead of features?

Reply:
I think we would want to keep the features tag for the Feature requests but maybe mention adding the selfhost tag as well if it is a feature for selhost. I think that kind of language would be most useful in the actual feature template which I can modify on forem.dev.

Reply:
Sounds good to me.